target/linux/ramips/dts/mt7621_ubiquiti_edgerouterx-sfp.dts

```diff
@@ -5,18 +5,18 @@
 / {
 	model = "UBNT-ERX-SFP";
 	compatible = "ubiquiti,edgerouterx-sfp", "mediatek,mt7621-soc";
+};
```

Comment:
Commit title too long, maybe remove Ubiquiti and second i2c:
"ramips: mt7621: use SoC I2C instead of bitbanged for ER-X-SFP"
You could also remove the mt7621 as well or instead of other stuff.